src/main/java/net/fabricmc/loom/ModExtension.java

```diff
@@ -0,0 +1,35 @@
+package net.fabricmc.loom;
+
+import org.gradle.api.Project;
+
+public class ModExtension {
+    private final Project project;
```

Comment:
Just need to fix the formatting on this file (tabs).

Reply:
intereesting.. tabs and spaces seem to be used interchangeably in the project
net.fabricmc.loom.util.MixinRefmapHelper for example is spaces ?